ambry-cloud/src/main/java/com/github/ambry/cloud/VcrReplicationManager.java

```diff
@@ -271,6 +267,11 @@ public long getRemoteReplicaLagFromLocalInBytes(PartitionId partitionId, String
     return -1;
   }
 
+  @Override
+  protected String getReplicaThreadName(String datacenterToReplicateFrom, int threadIndexWithinPool) {
+    return "Vcr" + ReplicaThread.getCanonicalReplicaThreadName(dataNodeId.getDatacenterName(), datacenterToReplicateFrom, threadIndexWithinPool);
```

Comment:
This can be simply:
`return "Vcr" + super. getReplicaThreadName(datacenterToReplicateFrom, threadIndexWithinPool);`
And then I think the canonical logic can be collapsed back into the superclass method.

Reply:
Sorry about the delay. Made the change.